compiler/qsc_frontend/src/compile.rs

```diff
@@ -35,13 +36,24 @@ use qsc_hir::{
 use std::{fmt::Debug, str::FromStr, sync::Arc};
 use thiserror::Error;
 
+bitflags! {
+    #[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
+    pub struct RuntimeCapabilityFlags: u32 {
```

Comment:
What benefit does it bring us to make this a `bitflags` rather than a struct with `bool` fields?

Reply:
A struct with bool fields with have a size of `n` bytes where `n` is the number of bools. The bitflags is much denser than that.

Reply:
Is there a particular perf reason that this needs to be small? There's basically only one instance of it that we're passing around at initialization. 

No objections to leaving it this way, just probing out of curiosity.